### DISCUSSION TITLE
Need Help Marker Text/Icon Overlapping Issue

### DISCUSSION THREAD

**Discussion by singhamritpal49** [uber/react-map-gl#1388] [Q&A]
($ Marker {:key index
:latitude (:Latitude location-single)
:longitude (:Longitude location-single)
:icon location -icon
:iconAllowOverlap true,
:textAllowOverlap false,
:textOptional true,
:offsetTop -40
:offsetLeft -20}
:data-place-id (:locationId location-single)}
(d/span {:class "text-fs-12"} (:Name location))
(d/img {:src location-icon :style {:width "40px" :height "40px" :margin "0 auto" :textAllowOverlap false :textOptional true }})

![Screen Shot 2021-03-03 at 3 23 51 PM](https://user-images.githubusercontent.com/48744353/109867610-b282eb00-7c34-11eb-9ad2-4d6232efc578.png)
![Screen Shot 2021-03-03 at 2 33 41 PM](https://user-images.githubusercontent.com/48744353/109867625-b7e03580-7c34-11eb-8826-024141fdc59a.png)

**Comment by eylonshm** [comment]
Hey, did you succeed ?
I'm trying to implement icon-allow-overlap and have some problems with it.
**How did you implement icon-allow-overlap, where did you add this code-line?**
Thank you :)
